### ISSUE TITLE
Some magics don't work

### ISSUE THREAD

**Issue by ddobrinskiy** [teaxyz/cli#423]
Can't find a pattern yet.

When zsh can't find a command, tea magic installs it, but then can't find it

After installing tea on a new Macbook Air M2, some magics work, while some don't.

Examples:

* `fd` and `exa` work out of the box
* `fzf` and `tree` don't work

```
$ pwd
/Users/david
$ fd fzf .tea
.tea/fishshell.com/v3.6.0/share/fish/completions/fzf.fish
.tea/github.com/junegunn/fzf/
.tea/github.com/junegunn/fzf/v0.38.0/fzf/
.tea/github.com/junegunn/fzf/v0.38.0/fzf/v0.38.0/bin/fzf
.tea/tea.xyz/var/pantry/projects/github.com/junegunn/fzf/
.tea/tea.xyz/var/www/github.com∕junegunn∕fzf-0.38.0+darwin+aarch64.tar.xz
$ fzf
tea: command not found: fzf
```

I am using zsh

**Comment by ddobrinskiy** [comment]
My `.zshrc` magic:

`test -d "$HOME/.tea" && source <("$HOME/.tea/tea.xyz/v*/bin/tea" --magic=zsh --silent)`

**Comment by getneil** [comment]
@ddobrinskiy the double nested prefix has been addressed by the fix in #420 .

**Comment by mxcl** [comment]
Fixed in 91bbd46a150903971d2d4339460f6f71e56bcc6d

0.24.8 building https://github.com/teaxyz/cli/actions/runs/4314029707

**Comment by mxcl** [comment]
Apologies, I should have been more thoughtful.
